### DISCUSSION TITLE
Audiobookshelf App Discussion

### DISCUSSION THREAD

**Discussion by advplyr** [advplyr/audiobookshelf-app#23] [Announcements]
Hello World!

**Comment by Budlyte** [comment]
Hiiiii advplyr!

Is there currently a way to "Mark Complete" an item while viewing in the mobile app?

I've been just enjoying your program and my audiobooks lately, instead of trying to break it.  :)

**Reply by p1xelll** [reply]
I started working on a minor redesign of the app. UI will be ready in a few days. All the features from the web will be added there. @advplyr will surely release an update then.

**Reply by p1xelll** [reply]
@Budlyte I would appreciate your feedback #33

**Comment by lukethehawk** [comment]
hello @advplyr ,
when I open the app there is the function that the app rewinds the track automatically after a long break?
For example Smart Audiobook Player has an automatic rewind that varies according to the length of the break.

Thanks for the work you do

**Reply by advplyr** [reply]
Hello. The function that rewinds after a break will only happen if the app is open and paused then unpaused. Once the app is closed and re-opened there will be no rewind. If that is a feature you are interested in you can open up a feature request.
